library/src/styles/fiori.css

```diff
@@ -1255,6 +1255,14 @@
 .asyncapi__channel-operation {
 }
 
+.asyncapi__channel-operation-description {
+  padding: 16px;
+}
+
+.asyncapi__channel-description {
+  padding: 16px;
+}
+
```

Comment:
Could you change it to:

```css
.asyncapi__channel-operation-description {
  padding: 16px 16px 0;
}

.asyncapi__channel-description {
  padding: 0 16px 16px;
}
```

and then description looks

![image](https://user-images.githubusercontent.com/20404945/94861949-9a663000-0438-11eb-96bc-9f5002e40aeb.png)

I don't like those empty space between toggles. Could you propagate my suggestion to your code? :)